roles/confluent.variables/tasks/main.yml

```
@@ -0,0 +1,26 @@
+---
+- name: Setting Host Variables on Secondary Supporting Kafka Cluster Hosts
+  set_fact: 
+    kafka_broker_listeners: "{{ kafka_broker_listeners }}"
+    ssl_enabled: "{{ ssl_enabled }}"
+    pkcs12_enabled: "{{ pkcs12_enabled }}"
+    ssl_mutual_auth_enabled: "{{ ssl_mutual_auth_enabled }}"
+    sasl_protocol: "{{ sasl_protocol }}"
+    kafka_broker_truststore_path: "{{ kafka_broker_truststore_path }}"
+    kafka_broker_truststore_storepass: "{{ kafka_broker_truststore_storepass }}"
+    kafka_broker_keystore_path: "{{ kafka_broker_keystore_path }}"
+    kafka_broker_keystore_storepass: "{{ kafka_broker_keystore_storepass }}"
+    kafka_broker_keystore_keypass: "{{ kafka_broker_keystore_keypass }}"
+    sasl_plain_users: "{{ sasl_plain_users }}"
+    sasl_scram_users: "{{ sasl_scram_users }}"
+    kerberos_kafka_broker_primary: "{{ kerberos_kafka_broker_primary }}"
+    kafka_broker_keytab_path: "{{ kafka_broker_keytab_path }}"
+    kafka_broker_kerberos_principal: "{{ kafka_broker_kerberos_principal  | default('kafka') }}"
+    kafka_broker_ldap_user: "{{ kafka_broker_ldap_user }}"
+    kafka_broker_ldap_password: "{{ kafka_broker_ldap_password }}"
+    mds_bootstrap_server_urls: "{{ mds_bootstrap_server_urls }}"
+    fips_enabled: "{{ fips_enabled }}"
+  delegate_to: "{{ item }}"
+  delegate_facts: true
+  with_items: 
```

```suggestion
  loop: 
```

also in the future we can just keep adding kafka groups to the list then? That will be slick!

---

Was using `with_items` for this very reason. Since a user may with to use a separate ansible group for each centralized cluster. Like RBAC, I'm going to want the most stable cluster which tends to be the main cluster where metrics I'm find with a smaller, maybe single node, cluster since the data is fine to be lost. `loop` would require the creation of a list within a single template. `with_items` lets us break it up.